apps/src/code-studio/pd/workshop_dashboard/workshop_summary_utils.js

```diff
@@ -0,0 +1,12 @@
+import {SubjectNames} from '@cdo/apps/generated/pd/sharedWorkshopConstants';
+
+export function useFoormSurvey(subject, lastSessionDate) {
+  // Local summer, CSF Intro, or CSP for returning teachings workshop
```

Comment:
Is "returning teachings workshop" the same as "WFRT" elsewhere?

Reply:
yeah wfrt stands for "workshop for returning teachers"

Reply:
realized it should have said teachers, not teachings, fixed